File: apps/udi-stub/src/main/java/no/nav/udistub/database/model/PersonNavn.java
package no.nav.udistub.database.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.Embeddable;

@AllArgsConstructor
@NoArgsConstructor
@Builder
@Getter
@Setter
@Embeddable
public class PersonNavn {
    private String fornavn;
    private String mellomnavn;
    private String etternavn;
}

File: apps/dolly-frontend/src/main/js/src/components/fagsystem/pdlf/visning/partials/Personnavn.js
import React from 'react'
import { TitleValue } from '~/components/ui/titleValue/TitleValue'

export const Personnavn = ({ data }) => {
	if (!data) return false
	const { fornavn, mellomnavn, etternavn } = data
	return (
		<>
			<TitleValue title="Kontaktperson fornavn" value={fornavn} />
			<TitleValue title="Kontaktperson mellomnavn" value={mellomnavn} />
			<TitleValue title="Kontaktperson etternavn" value={etternavn} />
		</>
	)
}
